app/lib/RepoUtil.scala

```diff
@@ -58,7 +58,10 @@ object RepoUtil extends Logging {
       } else {
         Files.createDirectories(gitdir.toPath.getParent)
         logger.info(s"Cloning new Git repo... $uri")
-        invoke(Git.cloneRepository().setBare(true).setDirectory(gitdir).setURI(uri)).getRepository
+        invoke(
+          Git.cloneRepository()
+            .setDepth(200) // the aim here is to reduce the memory load of big repos like guardian/frontend
```

Comment:
This is the one thing added by this change, if you ignore reformatting! We're only going to fetch the last 200 commits, rather than the full commit history - and hope that this reduces memory load.